client-native-image/README.md

```diff
@@ -0,0 +1,26 @@
+
+# Client Native Image
+
+This project is a simple example of how to create a GraalVM native image with a Ktor Client application.
+
+## Running the application
+
+To run the application, you can use the following command:
```

Comment:
Maybe it's worth to mention it's a console application that prints the contents of a web page so it's clear what to expect when running the application.